junction/proposals/forms.py

```diff
@@ -143,6 +143,12 @@ class ProposalForm(forms.Form):
         help_text="Links to your previous work like Blog, Open Source Contributions etc ...",
     )
 
+    is_first_time_speaker = forms.BooleanField(
+        label="First Time Speaker",
+        required=False,
+        help_text="Please tick, if you are a first time speaker"
```

Comment:
nitpick: does it sound good to have check instead of tick?

Reply:
"check" would sound better.

Reply:
I had added check initially, but thought it won't sound good. Let me update that.

Reply:
"mark" might also sound good.

Reply:
Added, please review 8154280889afde605c640298f249b6aeddebd194